addons/info/src/components/markdown/text.js

```diff
@@ -10,7 +10,12 @@ export function P(props) {
     ...baseFonts,
     fontSize: '15px',
   };
-  return <p style={style}>{props.children}</p>;
+
+  // <P> is oftentimes used as a parent element of
+  // <a> and <pre> elements, which is why <div>
+  // is used as the outputted element when parsing
+  // marksy content rather than <p>.
+  return <div style={style}>{props.children}</div>;
 }
 
 P.defaultProps = defaultProps;
```

Comment:
Maybe we should change the component name too? Looks like it's now called `P`, but renders a `DIV`.

Thanks to the comment we do know why though 👍 .

Reply:
Yeah it would make sense to change it, but in that case, do we go ahead and distinguish when the component is used as a wrapper and when it's just used as <p> tag? Or if not, what do we rename this to?